### PR TITLE
add possibility to write schemes inline (finishing #20)

### PR DESCRIPTION
**Pull request by matAlmeida**
Today I started using this lib and was wondering exactly if this feature was available and I saw that the PR #20 was on idle for about a month, so I made the changes asked by you guys. I tried to add something to the same PR but didn't found how.

### CODE REVIEW

**Comment by erunion** [comment]
Hey! Sorry for the delay on this one, can you rebase this branch with the latest master?

File: package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "commander": "^4.0.1",
     "fs-extra": "^0.30.0",
     "js-yaml": "^3.13.1",
-    "lodash": "^4.17.11",
+    "lodash": "^4.17.15",
```
Comment (erunion) [inline package.json]: Can you revert this lodash change, and the other changes to `package-lock.json`? We can handle that in a separate PR.

Comment (matAlmeida) [reply package.json]: didn't saw this files changing

Comment (erunion) [reply package.json]: np!

**Comment by erunion** [comment]
This is a good enough change for a major version bump so I've tagged this to 2.0.0. Thanks again!

**Pull request merged**